tests/e2e/performance/test_pvc_multi_clone_performance.py

```diff
@@ -0,0 +1,409 @@
+import datetime
+import logging
+import os
+import subprocess
+import tempfile
+import time
+
+import yaml
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version,
+    skipif_ocp_version,
+    E2ETest,
+    performance,
+)
+from ocs_ci.ocs import constants, exceptions
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.resources import pod
+from ocs_ci.utility.utils import ocsci_log_path
+
+log = logging.getLogger(__name__)
+
+
+@performance
+@skipif_ocp_version("<4.6")
+@skipif_ocs_version("<4.6")
+class TestPvcMultiClonePerformance(E2ETest):
+    """
+    Tests to measure PVC clones creation performance ( time and speed)
+    The test is supposed to create the maximum number of clones for one PVC
+    """
+
+    def test_pvc_multiple_clone_performance(
+        self,
+        interface_iterate,
+        teardown_factory,
+        storageclass_factory,
+        pvc_factory,
+        pod_factory,
+    ):
+        """
+        1. Creating PVC
+           PVS size is calculated in the test and depends on the storage capacity, but not less then 1 GiB
+           it will use ~75% capacity of the Storage, Min storage capacity 1 TiB
+        2. Fill the PVC with 70% of data
+        3. Take a clone of the PVC and measure time and speed of creation by reading start creation and end creation
+            times from relevant logs
+        4. Repeat the previous step number of times (maximal num_of_clones is 512)
+        5. Rrint all measured statistics for all the clones.
+
+        Raises:
+            StorageNotSufficientException: in case of not enough capacity on the cluster
+
+        """
+        num_of_clones = 512
+        if interface_iterate == constants.CEPHBLOCKPOOL:
+            num_of_clones = 512  # bz_1896831
```

Comment:
This always results in num_of_clones = 512

Reply:
@Avilir , @wusui  - I've kept it separate since previously there was a bug which resulted in different number of clones for RBD and CephFS to be successfully created. Removing it, since currently I've verified that it's possible to create 512 for each one of the interface types.